lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -244,37 +258,43 @@ putSignedTransaction (tx, witnesses) = do
 putWitness :: TxWitness -> Put
 putWitness witness =
     case witness of
-        PublicKeyWitness xPub (Hash sig) -> do
+        PublicKeyWitness _xPub (Hash sig) -> do
             -- Witness sum type:
-            --   * 1 for old address witnness scheme
-            --   * 2 for new address witness scheme
-            --   * 3 for account witness
+            --   * 0 for old address witnness scheme
+            --   * 1 for new address witness scheme
+            --   * 2 for account witness
+            --   * 3 for multisig
             putWord8 1
-            putByteString xPub
+            --putByteString xPub
```

Comment:
But then old address scheme won't work, right? (as it expects xPub in its format)